modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v6_0_0/util/ObjectFolderTable.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.internal.upgrade.v6_0_0.util;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Brian Wing Shun Chan
+ * @generated
+ * @see com.liferay.portal.tools.upgrade.table.builder.UpgradeTableBuilder
+ */
+public class ObjectFolderTable {
+
+	public static UpgradeProcess create() {
```

Comment:
Did you forget to call this?
I'm asking because I saw a similar situation with Carol's PR, and she called this class in the upgrade step.
Please, see: https://github.com/liferay-objects/liferay-portal/pull/2356/commits/0728475e5f0fb72b561fc57340b77519f1d0d71f#diff-e80f0b6b8785b5e743b6f298129dacf3e33a2cccab6fd4ec9856a03e47673195

Reply:
Hey @marcelabc 

I called this class in `getPreUpgradeSteps` [here](https://github.com/liferay-objects/liferay-portal/pull/2349/commits/99739023fc439502c3c6d3b8c33d8ae1b4e4ff51#:~:text=ObjectFolderTable.create()). We need to create the table before the upgrade started.